### ISSUE TITLE
Yesterday traffic eat a huge amount of money

### ISSUE THREAD

**Issue by vgoncharenko** [Arriven/db1000n#153]
Guys, do we have any coupons for AWS? yesterdays traffic eat huge amount of money.

Bandwidth$4,934.30
$0.000 per GB - data transfer in per month339.145 GB$0.00
$0.000 per GB - data transfer out under the monthly global free tier100.000 GB$0.00
$0.000 per GB - regional data transfer under the monthly global free tier0.025 GB$0.00
$0.070 per GB - next 100 TB / month data transfer out7,587.163 GB$531.10
$0.085 per GB - next 40 TB / month data transfer out40,960.000 GB$3,481.60
$0.090 per GB - first 10 TB / month data transfer out beyond the global free tier10,240.000 GB$921.60

**Comment by industral** [comment]
yes, write to them.. but I don't know how you'll explain that traffic.
They might forgive you that and might issue coupon to redeem.

**Comment by Arriven** [comment]
This would happen with other DDoS solutions running on clouds as well but we need to add a warning to instructions since not everyone might be aware that most big cloud providers bill you for traffic
@Amet13 @iamtodor can you guys do it?

**Comment by Amet13** [comment]
@Arriven yep

**Comment by Arriven** [comment]
I'll leave this issue as open for people to be aware at least until this war ends

**Comment by Amet13** [comment]
Maybe it would be great to label it somehow to highlight it from other issues (or pin it somehow at the top). _Not sure that GitHub allows issues pinning_

**Comment by Arriven** [comment]
I'll create a separate docker release for that in a minute but please use https://raw.githubusercontent.com/db1000n-coordinators/LoadTestConfig/main/config.adv.json
as your config on clouds to avoid this issue

**Comment by iamtodor** [comment]
relates to https://github.com/Arriven/db1000n/issues/54

**Comment by vgoncharenko** [comment]
AWS team have approved bill adjustments for my acc:
`We’ve approved a billing adjustment of $5,099.73 USD for charges on your March bill, which has been applied as a credit to your account. This credit will automatically apply towards this month’s usage and can be referenced in your Account and Billing Console.`

**Comment by industral** [comment]
congrats!

**Comment by khardho120777** [comment]
bahala kayo sa buhay mo problema nyu na yan masstress lang ako sainyo
